### ISSUE TITLE
PyCharm Robot Framework plugin renders PyCharm unusable. 

### ISSUE THREAD

**Issue by aaltat** [robocorp/robotframework-lsp#243]
**Describe the bug**
After installing the Robot Framework plugin, PyCharm is so slow, that it looks like it has crashed. 

**To Reproduce**
1. Install plugin 
2. Restart PyCharm
3. PyCharm comes unresponsive, menus and stuff opens after waiting a minute or something similar. 

**Expected behavior**
PyCharm should not suffer performance hit which would be so visible for the users. 

**Versions:**
 - OS: Windows 10
 - Robot Framework Version; 3.2
 - Robot Framework Language Server Version: Latest from CI two days ago. 
 - Client Version PyCharm 2020.3

**Comment by fabioz** [comment]
Thanks for the report.

Do you think you can give instructions on how to reproduce that?

I guess it's probably related to the project contents you're working with... is the project you're working on open source (i.e.: something I can get here locally to reproduce)?

**Comment by fabioz** [comment]
Note: I actually fixed 2 issues:

- the server stderr is now being consumed (which should fix a halting issue when communication to the shell would be interrupted).
- an option to do a lightweight version of workspace symbols which will only analyze open files was added. It should be turned on for big workspaces -- such as the RobotFramework project itself (note: it needs to be turned on in the preferences in the `robot.workspaceSymbolsOnlyForOpenDocs` setting).
